test/unit/plugin_test.rb

```diff
@@ -424,6 +424,25 @@ def test_hosts_controller_action_scope_added_to_local
     assert_equal 2, scopes.count
   end
 
+  def test_add_resource_permissions_to_core
+    Foreman::Plugin.register :test_plugin do
+      add_resource_permissions_to_core ["Test::Resource"], :except => [:create_test]
+    end
+    manager = Role.find_by :name => "Manager"
+    viewer = Role.find_by :name => "Viewer"
+    assert_equal 2, manager.permissions.where(:resource_type => "Test::Resource").count
```

Comment:
Why does this count go up by 2 only? I would have assumed 3 would have been added (view, edit and delete).